Assets/Scripts/Layers/Properties/PropertyPanels/MTLImportPropertySectionInstantiator.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using Netherlands3D.Twin.Layers;
+using Netherlands3D.Twin.Layers.Properties;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    public class MTLImportPropertySectionInstantiator : MonoBehaviour, IPropertySectionInstantiator
+    {
+        [SerializeField] private MTLImportPropertySection propertySectionPrefab; 
+        public MTLImportPropertySection PropertySection { get; private set; }
+
+        public void AddToProperties(RectTransform properties)
+        {
```

Comment:
puur om even te dubbel checken hier of de PropertySection == null?

Reply:
de PropertySection is hier null ja, maar die instantiate je en dan is die niet meer null, of bedoel je dat niet?

Reply:
nee meer van als ie al bestond hoef je hem niet opnieuw te instantiaten maar ik ken de flow niet goed genoeg hier

Reply:
als je het property paneel opent/sluit wordt hij opnieuw gemaakt/gedestroyed, dus dat kan niet